docs/modules/user-guide/partials/con_using-top-level-attribute.adoc

```diff
@@ -0,0 +1,38 @@
+[id="con_using-top-level-attribute_{context}"]
+= Using top-level attribute
+
+[role="_abstract"]
+
+This section explains how to use top-level attribute in devfiles in order to reuse global attributes in different sections of the same devfile.
+
+.Devfile variable
+
+The devfile 2.0 specification includes global variables, which defines key-value pairs.
+
+```yaml
+variables:
+  my-var: "some value"
+```
+
+These variables can be referenced in non-key string fields within the devfile using `{{my-var}}`. It is used to template different versions of a devfile by setting only the variable field.
```

Comment:
I would put this content _before_ the yaml example. So that way, it's the structure: 1) context. 2) example. 

Also, when you can, avoid passive voice and be user focused instead. What do you think of this rephrase?: 

`Use `{{my-var}}` to reference the variables in non-string fields within the devfile. When you set only the variable field, you can make different template versions of your devfile.`